logfire-api/logfire_api/_internal/baggage.pyi

```diff
@@ -0,0 +1,39 @@
+from _typeshed import Incomplete
+from collections.abc import Iterator
+from contextlib import contextmanager
+
+__all__ = ['get_baggage', 'set_baggage']
+
+get_baggage: Incomplete
+
+@contextmanager
+def set_baggage(**bag: str) -> Iterator[None]:
+    '''Context manager that attaches key/value pairs as OpenTelemetry baggage to the current context.
+
+    All values in `bag` must be strings, as OpenTelemetry baggage only supports string values.
+
+    OpenTelemetry baggage is a way to propagate metadata across service boundaries in a distributed system, and is
+    included in headers of outgoing requests for which context propagation is configured.
+    This is intended to be used to propagate arbitrary context (like user ids, task ids, etc.) down to all nested spans.
+
+    Baggage is not _automatically_ converted into attributes on descendant spans, but this is a common usage pattern.
+    If you want baggage to be converted into attributes, use `logfire.configure(add_baggage_to_attributes=True)`.
```

Comment:
```suggestion
    Baggage is not _automatically_ converted into attributes on descendant spans, but this is a common usage pattern.
    If you want baggage to be converted into attributes, use `logfire.configure(add_baggage_to_attributes=True)`.
    This is the default.
    To disable this set `logfire.configure(add_baggage_to_attributes=False)`
```

Reply:
ignore pyi files

https://github.com/pydantic/logfire/pull/1128#discussion_r2135493755

Reply:
This wording doesn't exist on the real function. These files weren't supposed to be generated and I'm not updating them each time.